src/network.tf

```diff
@@ -416,7 +416,7 @@ data "azuread_application" "vpn_app" {
 }
 
 module "vpn" {
-  source = "git::https://github.com/pagopa/azurerm.git//vpn_gateway?ref=v1.0.36"
+  source = "git::https://github.com/pagopa/azurerm.git/vpn_gateway?ref=vpvgw-diagnosticsettings"
```

Comment:
```suggestion
  source = "git::https://github.com/pagopa/azurerm.git//vpn_gateway?ref=vpvgw-diagnosticsettings"
```

Reply:
pushed with the latest module version tag